graspy/plot/__init__.py

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) Microsoft Corporation and contributors.
+# Licensed under the MIT License.
+
```

Comment:
note this file directory is graspy

Reply:
I have both the changes in the grapsy and graspologic directories. I am confused by what you mean by the comment above